webpay/pay/tests/test_tasks.py

```diff
@@ -392,7 +392,7 @@ def setUp(self):
                             'icons': {'64': 'http://app/i.png'},
                             'name': 'Virtual Sword',
                             'description': 'A fancy sword'}}}
-        self.prices = {'prices': [{'price': 1, 'currency': 'EUR'}]}
+        self.prices = {'prices': [{'price': '0.89', 'currency': 'EUR'}]}
```

Comment:
it is posted to solitude as a string but the form field is of type Decimal so it gets converted to a decimal: https://github.com/mozilla/solitude/pull/317/files#diff-7d54df5fa0a0a174145b48623082fb09R83